homeassistant/components/habitica/coordinator.py

```diff
@@ -56,7 +56,14 @@ async def _async_update_data(self) -> HabiticaData:
         try:
             user_response = await self.api.user.get()
             tasks_response = await self.api.tasks.user.get()
-            tasks_response.extend(await self.api.tasks.user.get(type="completedTodos"))
+            tasks_response.extend(
+                [
+                    {**task, "id": task.get("id", task["_id"])}
```

Comment:
Key order matters, so this will do the same too. Not needed optimization for now, as you'll be reverting ths when API is fixed.
```suggestion
                    {"id": task["_id"], **task}
```